packages/php/blueprint/src/Cli/ImportCli.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Automattic\WooCommerce\Blueprint\Cli;
+
+use Automattic\WooCommerce\Blueprint\CliResultFormatter;
+use Automattic\WooCommerce\Blueprint\ImportSchema;
+
+class ImportCli {
+	private $schema_path;
+	public function __construct($schema_path) {
+		$this->schema_path = $schema_path;
+	}
+
+	public function run($optional_args) {
+	    $blueprint = ImportSchema::create_from_file($this->schema_path);
+		$results = $blueprint->import();
+
+		$result_formatter = new CliResultFormatter($results);
+		$is_success = $result_formatter->is_success();
+
+		if (isset($optional_args['show-messages'])) {
+			$result_formatter->format($optional_args['show-messages']);
+		}
+
+		$is_success && \WP_CLI::success("$this->schema_path imported successfully");
+		!$is_success && \WP_CLI::error("Failed to import $this->schema_path. Run with --show-messages=all to debug");
```

Comment:
I don't have a strong preferences on this TBH. Since we use this pattern in JS often, I figured I should use it in PHP as well. Do you have any reason why you don't prefer this approach?

Reply:
Maybe nitpicky but a couple of things:
1. This pattern isn't widely used in PHP, when I first read it I was confused for a few seconds
2. I think it's also slightly harder to read 

Even more nitpicky:
1. We are doing the condition check twice - wasting CPU cycle 😆